### PR TITLE
class02_hw3-jiseung0630

### PR DESCRIPTION
**Pull request by jiseung0630**

### CODE REVIEW

File: class02/homework/jiseung0630/hw3/hw03_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+./splitted_dataset/val: 51
+./splitted_dataset/train/<O>: 26
+./splitted_dataset/train/<X>: 25
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S|1.0|  |0:00:28.156916|64|0.0071|
+|MobileNet-V3-large-1x|1.0|  | 0:00:09.828827|64|0.0058|
```
Comment (JerryJLee) [inline class02/homework/jiseung0630/hw3/hw03_template.md:20]: 모델도 2개 뿐이고 FPS 측정도 실패했군요..
